test/jdk/sun/net/ext/ExtendedSocketOptionsTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
```

Comment:
Tests do not need - nor should they have - the "Classpath" exception - please see Copyright notice of other tests in the vicinity.

Reply:
Thank you for catching this. I've fixed it in the latest update of the PR.